develop/toolkit/api-libraries/polkadot-js-sdk.md

```diff
@@ -1,4 +1,112 @@
 ---
-title: Polkadot.js SDK
-description: TODO
----
\ No newline at end of file
+title: Polkadot.js API
+description: Interact with Polkadot SDK-based chains easily using the Polkadot.js API. Query chain data, submit transactions, and more via JavaScript or Typescript.
+---
+
+# Polkadot.js API
+
+## Introduction
+
+The [Polkadot.js API](https://github.com/polkadot-js/api){target=\_blank} uses JavaScript/Typescript to interact with Polkadot SDK-based chains. It allows you to query nodes, read chain state, and submit transactions through a dynamic, auto-generated API interface.
+
+### Dynamic API Generation
+
+Unlike traditional static APIs, the Polkadot.js API generates its interfaces automatically when connecting to a node. Here's what happens when you connect:
+
+1. The API connects to your node
+2. It retrieves the chain's metadata
+3. Based on this metadata, it creates specific endpoints in the format - `api.<type>.<module>.<section>`
+
+### Available API Categories
+
+You can access three main categories of chain interactions:
+
+1. **[Runtime Constants](https://polkadot.js.org/docs/api/start/api.consts){target=\_blank}** (`api.consts`)
```

Comment:
```suggestion
- **[Runtime constants](https://polkadot.js.org/docs/api/start/api.consts){target=\_blank}** (`api.consts`)
```

These don't need to be in an ordered list as it's not required they be accessed sequentially